ote/src/cljs/ote/views/rental.cljs

```diff
@@ -27,33 +28,176 @@
+    :type :table
+    :table-fields [{:name ::t-service/price-per-unit
+                    :type :number
+                    :currency? true
+                    :style {:width "100px"}
+                    :input-style {:text-align "right" :padding-right "5px"}}
+
+                   {:name ::t-service/unit
+                    :type :string}] 
+    :delete? true
+    :add-label (tr [:buttons :add-new-price-class])}
+   ))
+
+(defn vehicle-prices [update-form! data]
+  (reagent/with-let [open? (reagent/atom false)]
+    [:div
+     [ui/flat-button {:label (tr [:rentals-page :open-rental-prices])
+                      :on-click #(reset! open? true)}]
+     [ui/dialog
+      {:open @open?
+       :auto-scroll-body-content true
+       :title (tr [:price-dialog :header-dialog])
+       :actions [(reagent/as-element
+                  [ui/flat-button {:label (tr [:buttons :close])
+                                   :on-click #(reset! open? false)}])]}
+      [form/form {:update! update-form!
+                  :name->label (tr-key [:field-labels :rentals]
+                                       [:field-labels :transport-service]
+                                       [:field-labels])}
+       ;; [(assoc-in (ts-common/service-hours-group) [:options :card?] false)]
+       [(assoc-in (price-group) [:options :card?] false)]
+       data]]
+     ]))
+
+(defn vehicle-group []
+  (form/group
+   {:label (tr [:rentals-page :header-vehicles])
+    :columns 3}
+   
+   {:name ::t-service/rental-vehicles
+    :type :table
+    :table-fields [{:name ::t-service/vehicle-type
+                    :type :string}
+                   {:name ::t-service/license-required
+                    :type :string}
+                   {:name ::t-service/minimum-age
+                    :type :number}
+                   {:name :price-group
+                    :label (tr [:field-labels :rentals ::t-service/vehicle-prices])
+                    :type :component
+                    :component (fn [{:keys [update-form! data]}]
+                                 [vehicle-prices update-form! data])
+                    }]
+    :delete? true
+    :add-label (tr [:buttons :add-new-vehicle])})
+  )
+
 (defn accessibility-group []
   (form/group
-   {:label (tr [:rentals-page :header-other-services-and-accessibility])
+   {:label (tr [:rentals-page :header-accessibility])
     :columns 3
     :layout :row}
 
-   {:name        ::t-service/additional-services
-    :type        :multiselect-selection
-    :show-option (tr-key [:enums ::t-service/additional-services])
-    :options     t-service/additional-services}
+   {:name        ::t-service/guaranteed-vehicle-accessibility 
+    :help (tr [:form-help :guaranteed-vehicle-accessibility])
+    :type        :checkbox-group
+    :show-option (tr-key [:enums ::t-service/vehicle-accessibility])
+    :options     t-service/rental-vehicle-accessibility
+    :full-width? true
+    :container-style (merge style-form/half-width
```

Comment:
Check these.
These can work in mobile devices.
Change to :container-class col-md-6 tms